HISTORY.md

```diff
@@ -1,3 +1,10 @@
+# Release 0.6
```

Comment:
Project.toml probably needs a matching update of the version number.

You could release this is 0.5.1 if you want, since, as far as I understand, it doesn't break any existing code that uses AdvancedVI. It's not wrong to use 0.6 though if you think the change is major enough to warrant it.

Reply:
We do export more symbols though. So isn't it supposed to be a minor release?